test/csuite/wt2853_perf/main.c

```diff
@@ -44,14 +44,19 @@ static void *thread_insert(void *);
 static void *thread_get(void *);
 
 #define BLOOM false
-#define MAX_GAP 7
+#define SHOW_GAP 5
+#define MAX_GAP 25
```

Comment:
Please alphabetize. I suggest renaming as `GAP_MAX` and `GAP_SHOW` so they sort together.